lib/auth.js

```diff
@@ -231,6 +234,47 @@ module.exports = new Model({
     }.bind(this));
   },
 
+  signInWithOAuthRedirect: function (args, window) {
+    // Check to see if we're running in the browser or not
+    if (typeof window === 'undefined') {
```

Comment:
`window` is declared as an argument, so you can compare it to `undefined` directly.